packages/card/src/Card.tsx

```diff
@@ -1,28 +1,55 @@
 import React from 'react';
+import { transparentize } from 'polished';
 import PropTypes from 'prop-types';
 import { css, cx } from '@leafygreen-ui/emotion';
 import { uiColors } from '@leafygreen-ui/palette';
 import Box, { ExtendableBox } from '@leafygreen-ui/box';
 
+const baseBoxShadow = `0 4px 10px -4px ${transparentize(0.7, uiColors.black)}`;
+const hoverBoxShadow = `0 2px 6px -2px ${transparentize(0.4, uiColors.black)}`;
+const focusBoxShadow = '0 0 0 3px #9dd0e7';
+
 const containerStyle = css`
+  position: relative;
   background-color: white;
   border: 1px solid ${uiColors.gray.light2};
-  border-radius: 4px;
-  box-shadow: 0px 4px 10px -4px ${uiColors.gray.light1};
+  border-radius: 7px;
+  box-shadow: ${baseBoxShadow};
   transition: border 300ms ease-in-out, box-shadow 300ms ease-in-out;
+`;
+
+const clickableStyle = css`
+  cursor: pointer;
+
+  &:focus {
+    outline: none;
+    box-shadow: ${baseBoxShadow}, ${focusBoxShadow};
+  }
 
   &:hover {
     border: 1px solid ${uiColors.gray.light2};
-    box-shadow: 0px 3px 6px -2px ${uiColors.gray.base};
+    box-shadow: ${hoverBoxShadow};
+
+    &:focus {
+      box-shadow: ${hoverBoxShadow}, ${focusBoxShadow};
+    }
   }
 `;
 
 interface CardProps {
   className?: string;
+  onClick?: React.MouseEventHandler;
 }
 
 const Card: ExtendableBox<CardProps> = ({ className, ...rest }: CardProps) => {
-  return <Box className={cx(containerStyle, className)} {...rest} />;
+  const clickable = rest.onClick !== undefined;
```

Comment:
To account for links, we should probably also check for the existence of `href` or `to` (for in cases like React router links being used with the `as` prop).

Reply:
I think `href` makes sense, but it's odd for the component's behavior to depend on a specific routing library's interface. This could lead to a non react-router component that happens to have a `to` prop unexpectedly getting the clickable behavior applied. I think I'd rather expose a prop that allows the consumer to explicitly specify if the card is clickable, e.g. `clickable?: boolean;`.

Reply:
I dig adding that prop. If that prop isn't passed, it can still fall back to appearing clickable if `onClick` or `href` are passed as well. It also has the added benefit of being a way to disable the behavior in the off-chance you need to have an 'onClick' handler while not having the visual affordance.